src/components/sds/Icon/index.tsx

```diff
@@ -0,0 +1,2421 @@
+  Sliders04,
+  Snowflake01,
+  Snowflake02,
+  SpacingHeight01,
+  SpacingHeight02,
+  SpacingWidth01,
+  SpacingWidth02,
+  Speaker01,
+  Speaker02,
+  Speaker03,
+  Speedometer01,
+  Speedometer02,
+  Speedometer03,
+  Speedometer04,
+  Square,
+  Stand,
+  Star01,
+  Star02,
+  Star03,
+  Star04,
+  Star05,
+  Star06,
+  Star07,
+  Stars01,
+  Stars02,
+  Stars03,
+  StickerCircle,
+  StickerSquare,
+  StopCircle,
+  StopSquare,
+  Stop,
+  Strikethrough01,
+  Strikethrough02,
+  StrikethroughSquare,
+  Subscript,
+  SunSetting01,
+  SunSetting02,
+  SunSetting03,
+  Sun,
+  Sunrise,
+  Sunset,
+  SwitchHorizontal01,
+  SwitchHorizontal02,
+  SwitchVertical01,
+  SwitchVertical02,
+  Table,
+  Tablet01,
+  Tablet02,
+  Tag01,
+  Tag02,
+  Tag03,
+  Target01,
+  Target02,
+  Target03,
+  Target04,
+  Target05,
+  Telescope,
+  TerminalBrowser,
+  TerminalCircle,
+  TerminalSquare,
+  Terminal,
+  TextInput,
+  Thermometer01,
+  Thermometer02,
+  Thermometer03,
+  ThermometerCold,
+  ThermometerWarm,
+  ThumbsDown,
+  ThumbsUp,
+  Ticket01,
+  Ticket02,
+  Toggle01Left,
+  Toggle01Right,
+  Toggle02Left,
+  Toggle02Right,
+  Toggle03Left,
+  Toggle03Right,
+  Tool01,
+  Tool02,
+  Train,
+  Tram,
+  Transform,
+  Translate01,
+  Translate02,
+  Trash01,
+  Trash02,
+  Trash03,
+  Trash04,
+  TrendDown01,
+  TrendDown02,
+  TrendUp01,
+  TrendUp02,
+  Triangle,
+  Trophy01,
+  Trophy02,
+  Truck01,
+  Truck02,
+  Tv01,
+  Tv02,
+  Tv03,
+  Type01,
+  Type02,
+  TypeSquare,
+  TypeStrikethrough01,
+  TypeStrikethrough02,
+  Umbrella01,
+  Umbrella02,
+  Umbrella03,
+  Underline01,
+  Underline02,
+  UnderlineSquare,
+  Upload01,
+  Upload02,
+  Upload03,
+  Upload04,
+  UploadCloud01,
+  UploadCloud02,
+  UsbFlashDrive,
+  User01,
+  User02,
+  User03,
+  UserCheck01,
+  UserCheck02,
+  UserCircle,
+  UserDown01,
+  UserDown02,
+  UserEdit,
+  UserLeft01,
+  UserLeft02,
+  UserMinus01,
+  UserMinus02,
+  UserPlus01,
+  UserPlus02,
+  UserRight01,
+  UserRight02,
+  UserSquare,
+  UserUp01,
+  UserUp02,
+  UserX01,
+  UserX02,
+  Users01,
+  Users02,
+  Users03,
+  UsersCheck,
+  UsersDown,
+  UsersEdit,
+  UsersLeft,
+  UsersMinus,
+  UsersPlus,
+  UsersRight,
+  UsersUp,
+  UsersX,
+  Variable,
+  VideoRecorderOff,
+  VideoRecorder,
+  Virus,
+  Voicemail,
+  VolumeMax,
+  VolumeMin,
+  VolumeMinus,
+  VolumePlus,
+  VolumeX,
+  Wallet01,
+  Wallet02,
+  Wallet03,
+  Wallet04,
+  Wallet05,
+  WatchCircle,
+  WatchSquare,
+  Waves,
+  Webcam01,
+  Webcam02,
+  WifiOff,
+  Wifi,
+  Wind01,
+  Wind02,
+  Wind03,
+  XCircle,
+  XClose,
+  XSquare,
+  X,
+  Youtube,
+  ZapCircle,
+  ZapFast,
+  ZapOff,
+  ZapSquare,
+  Zap,
+  ZoomIn,
+  ZoomOut,
+};
+
+export interface IconProps {
+  name?: keyof typeof Icons;
+  circle?: boolean;
+  size?: number;
+  color?: string;
+  testID?: string;
+}
+
+const CircleContainer = styled.View`
+  width: ${({ size }: { size: number }) => size * 1.5}px;
+  height: ${({ size }: { size: number }) => size * 1.5}px;
```

Comment:
I think we should wrap those with `px()` instead of the inline px string so it resizes accordingly.